### ISSUE TITLE
What do you think about diffgrad?

### ISSUE THREAD

**Issue by hadaev8** [JRC1995/DemonRangerOptimizer#1]
This is a greate repo, my respect.

**Comment by JRC1995** [comment]
Thanks. 

I haven't read the paper on diffgrad.  Abstract looks interesting.

**Comment by hadaev8** [comment]
Also, what do you think about weight decay in style of adamw and gradient norming? Will I break anything with gradient norming?

**Comment by JRC1995** [comment]
It should be good.
